imjoy/runners/subprocess.py

```diff
@@ -24,10 +24,11 @@
     run_process,
 )
 
-DEFAULT_REQUIREMENTS_PY2 = ["numpy", "python-socketio[client]"]
-DEFAULT_REQUIREMENTS_PY3 = ["numpy", "python-socketio[client]", "janus"]
-REQ_PSUTIL = ["psutil"]
-REQ_PSUTIL_CONDA = ["conda:psutil"]
+DEFAULT_REQUIREMENTS = ["numpy==1.16.4", "python-socketio[client]==4.0.3"]
+DEFAULT_REQUIREMENTS_PY2 = DEFAULT_REQUIREMENTS
+DEFAULT_REQUIREMENTS_PY3 = [*DEFAULT_REQUIREMENTS, "janus==0.4.0"]
+REQ_PSUTIL = ["psutil==5.6.2"]
+REQ_PSUTIL_CONDA = ["conda:psutil=5.6.2"]
```

Comment:
I don't have a strong opinion here, but since `REQ_PSUTIL_CONDA` is a fallback package, maybe we should also leave out the version, any existing version should be fine?